mysql-test/suite/rocksdb/t/issue7891.test

```diff
@@ -0,0 +1,13 @@
+--source include/have_rocksdb.inc
```

Comment:
It looks like tese cases are 1:1 taken from pstress run.
Please:
1. Fix formatting (unneeded whitespaces, etc)
2. table/column/index naming
3. Are tables structures minimal? I've found this case to be minimal:
```
CREATE TABLE tt_5 (
	t0 TEXT,
	f3 FLOAT,
	g4 VARCHAR(4),
	INDEX tt_5i3(f3, t0(2), g4 ASC))
	ENGINE=RocksDB ;
INSERT INTO tt_5  ( t0 ,f3 ,g4 ) VALUES( 'WID5rpEI0cHsvwPtW77IEGYMAhVHBJKTVwMgUzaluvLR1se4uo3hZblvSW0I7xvcq9qa8H9ThW',  0.00, 'abcd' ) ;
UPDATE tt_5 SET g4 = 'ABCD' WHERE f3 = 0.00 ;
DROP TABLE tt_5;
```
Maybe we can narrow it down further?
4. Please change the name of the test to be meaningful